versions/3.1.1.md

```diff
@@ -2346,6 +2346,13 @@ While composition offers model extensibility, it does not imply a hierarchy betw
 To support polymorphism, the OpenAPI Specification adds the `discriminator` keyword.
 When used, the `discriminator` indicates the name of the property that hints which schema definition is expected to validate the structure of the model.
 
+###### Generic (Template) Data Structures
+
+Implementations MAY support defining generic or template data structures using JSON Schema's dynamic referencing feature:
+
+* `$dynamicAnchor` identifies a set of possible schemas (including a default placeholder schema) to which a `$dynamicRef` can resolve
+* `$dynamicRef` resolves to the first matching encountered on its path from the schema entry point to the reference, as described in the JSON Schema specification
```

Comment:
```suggestion
* `$dynamicRef` resolves to the first matching `$dynamicAcnhor` encountered on its path from the schema entry point to the reference, as described in the JSON Schema specification
```

This has to be probably the most concise explanation of dynamic references that I've ever seen. 👍

Reply:
@gregsdennis it only took me three and a half years to figure out how to boil it down... 😅

Reply:
Although you typo'd `$dynamicAcnhor` so I'll incorporate this separately as I don't know how to edit a suggestion.

Reply:
Fixed proposal